src/writer.jl

```diff
@@ -228,6 +245,11 @@ function render(doc::Documenter.Document, settings::MarkdownVitepress=MarkdownVi
         touch(config_path)
     end
 
+    objects_inv = joinpath(builddir, "final_site", "objects.inv")
```

Comment:
@asinghvi17 I'm having trouble understanding the folders used in DocumenterVitepress' build process. What is the correct place to write the `objects.inv` file to, to ensure that it ultimately gets deployed in the root of a particular version of the documentation? I guess `builddir`, maybe? Ideally, it would also exist in whatever `DocumenterVitepress.dev_docs("build")` renders (not `builddir`, apparently). The `final_site` subfolder seems wrong: the file then gets deleted later on (not sure where or why)

Reply:
It should be `build/.documenter/`. I usually send static data to `build/.documenter/assets/` or `build/.documenter/public`. 

Vitepress `build`s the site taking the files from `.documenter` and that built we send to `final_site`.

Reply:
see: https://github.com/LuxDL/DocumenterVitepress.jl/blob/c2fc02e4fab6d819c63cbac22bbfb0994c0b4486/src/writer.jl#L159 for other examples copying files.

Reply:
Most likely it should go to builddir/config.md_output_path/public, which will make it accessible at the top level after Vitepress builds.

Reply:
```
objects_inv = joinpath(builddir, settings.md_output_path, "public", "objects.inv")
```

seems to work. It makes the file show up in `build/final_site`, and, with the option ` clean_md_output = true`, in `build`.